sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -103,7 +107,7 @@ def __init__(self, operation_name, step_name, consumers, counter_factory,
     # We must do this manually as we don't have a spec or spec.output_coders.
     self.receivers = [
         operations.ConsumerSet(self.counter_factory, self.step_name, 0,
-                               next(consumers.itervalues()),
+                               next(itervalues(consumers)),
```

Comment:
I used itervalues based on the feedback of @tvalentyn [here](https://github.com/apache/beam/pull/5373#discussion_r195550396):

> As we learned in #5053, Cython can translate iteration through items() without an intermediate object given a directive to interpret the code using Python3 semantics. However when the code is not cythonized, I would prefer to use future.util.iteritems() avoid impact on efficiency for Python 2. It's probably ok to use items() in tests, or when it's clear that the collection is small in size.

Similar to iteritems, I am using itervalues here to avoid efficiency losses in Python2